.github/workflows/publish-pre-release.yml

```diff
@@ -19,38 +19,40 @@ jobs:
     steps:
 
     - uses: actions/checkout@v1
-      
-    - name: Setup .NET Core
-      uses: actions/setup-dotnet@v1
-
-    - name: Generate build number
-      uses: zyborg/gh-action-buildnum@v1.1.0 
-      with:
-        gist_token: ${{ secrets.GIST_TOKEN }}
-        set_env: true
+
+    # TEMPORARY DISABLED - Do we want to continue with this?
+
+    # - name: Setup .NET Core
+    #   uses: actions/setup-dotnet@v1
+
+    # - name: Generate build number
```

Comment:
The question is how do we generate a build number?
This is for nuget but it wont effect the node agent version which is driven from assembly info

Reply:
There is no difference than before, I disabled the pre-release until we are actually using it.

Updating new version is done by editing <Version>1.0.4</Version> in "Directory.Build.props" under src folder. There is only one of those files, and the version number is only in one location. It will generate FileVersion, NuGet version etc. all from that single source.